controllers/reconcile.go

```diff
@@ -420,6 +426,76 @@ func (r *FrontendReconciliation) populateCacheBustContainer(j *batchv1.Job) erro
 	return nil
 }
 
+// populatePushCacheContainer adds the push cache container to the deployment
+func (r *FrontendReconciliation) populatePushCacheContainer(j *batchv1.Job) error {
+	configMap := &v1.ConfigMap{}
+	configMap.SetName("pushcache")
+	configMap.SetNamespace(r.Frontend.Namespace)
+
+	nn := types.NamespacedName{
+		Name:      "pushcache",
+		Namespace: r.Frontend.Namespace,
+	}
+	labels := r.FrontendEnvironment.GetLabels()
+	labler := utils.GetCustomLabeler(labels, nn, r.FrontendEnvironment)
+	labler(configMap)
+
+	configMap.SetOwnerReferences([]metav1.OwnerReference{r.Frontend.MakeOwnerReference()})
+
+	// Create the configmap with the Client if it doesn't already exist
+	if err := r.Client.Create(r.Ctx, configMap); err != nil {
+		if !k8serr.IsAlreadyExists(err) {
+			return err
+		}
+	}
+
+	pushCacheVolume := v1.Volume{
```

Comment:
If the pushcache errors, will it not clear the cache? important while we are migrating i suppose